test/performance/observed_concurency_test.go

```diff
@@ -112,8 +110,9 @@ func parseResponse(body string) (*event, *event, error) {
 
 // timeToScale calculates the time it took to scale to a given scale, starting from a given
 // time. Returns an error if that scale was never reached.
-func timeToScale(events []*event, start time.Time, desiredScale int) (time.Duration, error) {
+func timeToScale(events []*event, desiredScale int) (time.Duration, error) {
 	var currentConcurrency int
+	start := events[0].timestamp
```

Comment:
The scaled pods can be created on different nodes, which can also have different clock, so this measurement might not be accurate as well?
But this way must be better than before because the timestamps are recorded from one single cluster.

Reply:
Within same cluster there's NTP which most likely restricts the time drift to negligible values.

Reply:
I see, nice :)